### PR TITLE
Drug UpdateCommand and Exit/HelpCommand

### PR DESCRIPTION
**Pull request by nevinlim**
-implemented UpdateCommand and UpdateCommandParser for Drug
-implemented ExitCommand and HelpCommand for general 
-changed mainwindow icon
-updated UG (screenshot and some commands)

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -208,20 +224,22 @@ _Details coming soon ..._
...
+| **Delete drug by trade name** | `d delete -tn TRADE_NAME`<br> e.g., `d delete -tn Panadol Flu Max`                                                                                                                                                                             |
+| **Delete drug by index**      | `d delete -i INDEX`<br> e.g.,`d delete -i 7`                                                                                                                                                                                                   |
+| **Update drug storage count** | `d update -tn TRADE_NAME -sc +-VALUE`<br> e.g.,`d update -tn Panadol Flu Max -sc +90`                                                                                                                                                          |
+| **Exit**                      | `g exit`                                                                                                                                                                                                                                       |
```
Comment (cydtseng) [inline docs/UserGuide.md:244]: Great formatting! 👍

**Review by cydtseng — COMMENTED** [review]
Some nitpicks, but otherwise lgtm!

**Review by Emrysil — COMMENTED** [review]
some minor places to fix, others LGTM :))))

File: src/main/java/seedu/address/logic/parser/drugparser/UpdateCommandParser.java [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    @Override
+    public UpdateCommand parse(String userInput) throws ParseException {
+        ArgumentMultimap argumentMultimap = ArgumentTokenizer.tokenize(userInput, PREFIX_TRADE_NAME, PREFIX_UPDATE);
+        if (arePrefixesPresent(argumentMultimap, PREFIX_TRADE_NAME, PREFIX_UPDATE)) {
+            TradeName tradeName = ParserUtil.parseTradeName(argumentMultimap.getValue(PREFIX_TRADE_NAME).get());
+            String update = argumentMultimap.getValue(PREFIX_UPDATE).get();
+            boolean add;
+            if(update.charAt(0) == '-') {
+                add = false;
+            } else if (update.charAt(0) == '+') {
```
[the remark this answers is not in the document]
Comment (nevinlim) [reply src/main/java/seedu/address/logic/parser/drugparser/UpdateCommandParser.java:27]: thanks for suggestion, i'll do that

File: src/main/java/seedu/address/logic/commands/drugcommands/AddCommand.java [hunk 3]
```diff
@@ -22,25 +22,25 @@ public class AddCommand extends Command {
...
-            + PREFIX_SIDE_EFFECT + "nausea "
-            + PREFIX_SIDE_EFFECT + "unexplained bruising or bleeding "
-            + PREFIX_DIRECTION + "Adults and children over 12 years: 1-2 caplets taken every 4 to 6 hours. Not " +
+            + PREFIX_TRADE_NAME + " Panadol "
+            + PREFIX_ACTIVE_INGREDIENT + " Paracetamol "
+            + PREFIX_PURPOSE + " relieve pain "
+            + PREFIX_PURPOSE + " relieve fever "
+            + PREFIX_PURPOSE + " relieve headache "
+            + PREFIX_SIDE_EFFECT + " skin rash "
+            + PREFIX_SIDE_EFFECT + " swelling of the lips, tongue, throat or face "
```
[the remark this answers is not in the document]
Comment (nevinlim) [reply src/main/java/seedu/address/logic/commands/drugcommands/AddCommand.java:38]: yeah, good point. i'll fix it

File: src/main/java/seedu/address/logic/parser/drugparser/UpdateCommandParser.java [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+                add = true;
+            } else {
+                throw new ParseException("Unknown/missing symbol ('+' for addition and '-' for subtraction concatenated in front) for update value");
+            }
+            Integer value = Integer.parseInt(update.substring(1));
```
[the remark this answers is not in the document]
Comment (nevinlim) [reply src/main/java/seedu/address/logic/parser/drugparser/UpdateCommandParser.java:32]: thanks for the suggestion. i'll implement it

**Pull request merged**